app/konnect/declarative-config.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Managing Konnect Cloud with declarative config
+no_version: true
+---
+
+Manage entities in your {{site.konnect_saas}} org with declarative
+configuration. To do this, you need [decK v1.7.0 or later](/deck/).
+
+With declarative configuration, you can manage:
+* **All parts of a service:** Service versions, implementations, routes, and
+plugins.
+* **Dev Portal documents:** Specs and markdown files.
+* **Global configuration:** Use declarative config to manage global plugins and
+ consumers. Global plugins appear in {{site.konnect_saas}} as read-only.
```

Comment:
Global/shared is yet another gateway/Konnect divergence of terms, with "global" being the original gateway term. Good question though, I'll do some digging to see where the terms are used.

Can't do anything about consumers though, that's an object and UI term.

Reply:
And after more digging and helping out with some QA on shared config, I've realized that unfortunately, global and shared config are not the same thing. 

**Shared config:** Konnect config that applies to...everything that isn't services? This is unclear, because for plugins, _all plugins_ are listed in shared config, even the service/route ones. Otherwise, shared config is only for editing all the things that are not services or Dev Portal. Tl;dr, it's where they lumped in everything they couldn't categorize another way.

**Global plugin:** a plugin that applies to all services, routes, and consumers in the cluster.

So the term "global config" was a mistake -- I think only plugins can be global, from what I've been able to find.